Cli-CredentialHelper/Program.cs

```diff
@@ -105,6 +105,9 @@ internal static Version Version
         }
         private static Version _version;
 
+        // Storing OperationArguments here for the sake of Github ModalPrompts
+        private static OperationArguments operationArguments;
```

Comment:
Not a huge fan of making this a static global. Personally, I'd rather see the reference passed around to various method, which should make testing easier in the future (when I get to authoring more of those).